src/main/java/net/minecraftforge/common/extensions/IForgeBlock.java

```diff
@@ -920,4 +920,30 @@ default MaterialColor getMapColor(BlockState state, BlockGetter level, BlockPos
     {
         return defaultColor;
     }
+
+    /**
+     * Returns the factor by which jump height is multiplied for any entities inside this block or standing on this block (unless overridden by the block the entity is in).
```

Comment:
Make this multiline, javadocs are smart enough to know how to interpret it. My general statement is that it should wrap after half a screen.